client/components/navbar.js

```diff
@@ -6,7 +6,10 @@ import {logout} from '../store'
 
 const Navbar = ({handleClick, isLoggedIn}) => (
   <div>
-    <nav>
+    <nav className="d-flex flex-row-reverse bd-highlight">
+      <a href="/cart" className="p-2 bd-highlight">
```

Comment:
Links that link to other parts of the app should be a <Link> tag, not an <a> tag.